VirtoCommerce.Platform.Web/Scripts/app/systemInfo/blades/system-info.js

```diff
@@ -0,0 +1,56 @@
+angular.module('platformWebApp')
+    .controller('platformWebApp.systemInfoController', ['$scope', 'platformWebApp.validators', 'platformWebApp.dialogService', 'virtoCommerce.contentModule.contentApi', '$timeout', 'platformWebApp.bladeNavigationService', '$http', 'platformWebApp.diagnostics',
+        function ($scope, validators, dialogService, contentApi, $timeout, bladeNavigationService, $http, diagnostics) {
+            var blade = $scope.blade;
+
+            function stringify(data) {
+                return JSON.stringify(data, null, "\t");
+            }
+
+            blade.initializeBlade = function () {
+                diagnostics.getSystemInfo({}, function (results) {
+                    blade.isLoading = false;
+                    blade.title = "Platform Info";
```

Comment:
Why the text "Platform Info" is not localized here?

Reply:
This is fixed